attachments/views.py

```diff
@@ -60,8 +70,33 @@ def attach(request, session_id):
                     session.data = {key: value}
             session.save()
             return JsonResponse({'ok': True, 'file_name': f.name, 'file_size': f.size}, content_type=content_type)
+        except FileSizeException as ex:
+            return JsonResponse({'ok': False, 'error': unicode(ex)}, content_type=content_type)
+        except FileTypeException as ex:
+            return JsonResponse({'ok': False, 'error': unicode(ex)}, content_type=content_type)
         except VirusFoundException as ex:
-            logger.exception(str(ex))
+            now = datetime.datetime.now()
+            now = now.strftime('%m-%d-%Y  %H:%M:%S')
+            #request.user may not exist so set up user_prefix to use right prefix for messages on virus upload
+            user = getattr(request, 'user', None)
```

Comment:
I would just pass user into the template and let it decide, via something like `{{ user|default_if_none:"Some Guy" }}`. The less formatting in code the better.